inclusive-design/covid-assessment-centres
# feat: add linting (resolves #8)

## Description

Use [fluid-grunt-lint-all](https://www.npmjs.com/package/fluid-grunt-lint-all) to lint files in this repository. Also added a CI workflow to run the linting task at pushes and issuing pull requests.

## Steps to test

1. Run `npm run lint` in a terminal;
2. Linting should pass;
3. Review and test the Github workflow at `.github/workflows/ci.yml`;
4. The CI should pass.

**Expected behavior:** 

See above.

Resolves #8
